main.go

```diff
@@ -313,6 +328,12 @@ func main() {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(4)
 		}
+	case "var":
```

Comment:
I think you committed this to the wrong branch? Is this for a different PR?

Reply:
Thanks, I'll defer this until a future PR.